cmake/3rdParty/FindPIX.cmake

```diff
@@ -6,15 +6,14 @@
 #
 #
 
-file(TO_CMAKE_PATH "$ENV{ATOM_PIX_PATH}" ATOM_PIX_PATH_CMAKE_FORMATTED)
+if(LY_PIX_ENABLED)
+    file(TO_CMAKE_PATH "${LY_PIX_PATH}" PIX_PATH)
+    message(STATUS "PIX PATH ${PIX_PATH}")
```

Comment:
We should either remove the STATUS message or update it to mention that `"PIX has been enabled at path ${PIX_PATH}"`